.apm/instructions/dev-commands.instructions.md

```
@@ -5,15 +5,22 @@ applyTo: "**"
 
 ### Database migration
 
-Run migrations: `go run ./cmd/sippy migrate --database-dsn $SIPPY_DATABASE_DSN`
+The devcontainer has two databases: **seed** (`postgres`) and **prod-like** (`prodlike`), controlled by `SIPPY_DATA_MODE` (default: `seed`).
 
-If `SIPPY_DATABASE_DSN` is not set, use the dev default: `postgresql://postgres:password@localhost:5432/postgres`
+To migrate both databases:
 
-### Linting
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)